source/daplink/drag-n-drop/vfs_manager.c

```diff
@@ -174,6 +175,18 @@ void vfs_mngr_fs_remount(void)
     sync_unlock();
 }
 
+void vfs_mngr_fs_remount_abort(void)
```

Comment:
This could cause problems if called externally so it should have file static scope.